packages/common/src/auth/ability.ts

```diff
@@ -27,6 +27,7 @@ export type Subject =
   | 'notification'
   | 'reaction'
   | 'invite_code'
+  | 'payment'
```

Comment:
I think I see why auth tripped you up.
Adding it here simply adds it to the types that are allowed

But for `authorize()` not to throw, you also need something like this down around line 64
```
can('create', 'payment'),
```

That said, I think that still would've failed if they were logged out, so deleting the authorize() line was probably correct in this case